committeeoversightapp/models.py

```diff
@@ -94,10 +88,208 @@ def chair(self):
     def hide_rating(self):
         return CommitteeDetailPage.objects.get(committee=self.id).hide_rating
 
+    @property
+    def ratings(self):
+        return self.committeerating_set.all().order_by('-congress')
+
+    @property
+    def ratings_reverse(self):
```

Comment:
`ratings` / `ratings_reverse` imply that the default ratings ordering will be used, but you implement a custom sort in the properties. Is there a more descriptive way to name these properties, so ordering is self-evident? Maybe `ratings_in_congress_order` or something to that effect?